networks/canarynet-uat/assets.json

```diff
@@ -0,0 +1,53 @@
+{
```

Comment:
rename to stagenet
and we might also need localnet (matching the docker-compose env.)